src/test/java/com/keylane/MainTest.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package com.keylane;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class MainTest {
+    @Test void appHasAGreeting() {
+        Main classUnderTest = new Main();
+        assertNotNull(classUnderTest.getGreeting(), "app should have a greeting");
```

Comment:
This is not going to work. Method getGreeting not in main class

Reply:
Removed unused test.